move_group: "arm"
planner_id: "PilzCircular"
planning_time: 5.0
eef_step: 0.01
jump_threshold: 0.0
velocity_scaling: 0.1
acceleration_scaling: 0.1
goal_tolerance: 0.001
ik_solver: "trac_ik"
safety_limit_file: "config/safety_limit.yaml"
